### DISCUSSION TITLE
Question about data usage

### DISCUSSION THREAD

**Discussion by nikhilCad** [z-huang/music#640] [Q&A]
If I listen to a song, then close the app, and maybe next day listen to it again with mobile data on, will the app stream it again or play it frm cache? Assume I have not downloaded it.

Also is there any way to make it use Wifi data only, and stop on mobile data.

**Comment by siggi1984** [comment]
The app will then play the song from the cache.

**Comment by WirelessAlien** [comment]
> Also is there any way to make it use Wifi data only, and stop on mobile data.


Yes, you can restrict the app from using mobile data. There is a general setting in android where you can deny the mobile data or wifi data uses permission to any app, then the app will be able to use only which one you allow.
